### PR TITLE
Add Rate of change Percentage indicator

### PR DESCRIPTION
**Pull request by Elizaveta239**
The result of PyCon 2016 Sprints.

### CODE REVIEW

File: zipline/pipeline/factors/technical.py [hunk 1]
```diff
@@ -162,6 +162,36 @@ def compute(self, today, assets, out, close, volume):
...
+    """
+    Rate of change Percentage
+    ROC measures the percentage change in price from one period to the next.
+    The ROC calculation compares the current price with the price `n`
+    periods ago.
+    Formula for calculation: ((price - prevPrice) / prevPrice) * 100
+    price - the current price
+    prevPrice - the price n days ago, equals window length
+
+    **Default Inputs**: [USEquityPricing.close]
```
Comment (llllllllll) [inline zipline/pipeline/factors/technical.py]: This factor seems like it is sufficiently general, how do you feel about removing the defaults here?

File: tests/pipeline/test_technical.py [hunk 2]
```diff
@@ -174,3 +175,19 @@ def test_aroon_basic(self, lows, highs, expected_out):
...
+            window_length=10
+        )
+        today = pd.Timestamp('2014')
+        assets = np.arange(5, dtype=np.int64)
+
+        data = np.ones((10, 5))
+        data[0, :] = np.full((1, 5), 2.0)
+        out = np.zeros(data.shape[1])
+        rocp.compute(today, assets, out, data)
+        assert_equal(out, np.full((5,), -50.0))
```
Comment (llllllllll) [inline tests/pipeline/test_technical.py]: Thanks for the good test case. Could you add one or two more cases as a regression test?

**Comment by llllllllll** [comment]
ping @Elizaveta239, do you think you will have time to make the suggested changes, if not, we can finish this up for you.

**Comment by Elizaveta239** [comment]
@llllllllll Sorry for the delay, I'm going to do it this weekend.

**Comment by llllllllll** [comment]
No rush, I just wanted to check in.

**Comment by Elizaveta239** [comment]
@llllllllll Hi! Unfortunately I won't be able to implement the changes during the next 3 weeks. I think, the better idea will be if you make the changes. I'm very sorry for that.

**Comment by richafrank** [comment]
@Elizaveta239 I updated and merged your changes in with #1324, so I'm closing this PR.  Thanks so much for spending some time with us and for integrating this indicator!

**Pull request closed without merging**